src/Model/User.php

```diff
@@ -213,6 +215,10 @@ public static function getFirstAdmin(array $fields = [])
 	 */
 	public static function getOwnerDataById($uid, $check_valid = true)
 	{
+		if (!empty(self::$owner)) {
+			return self::$owner;
```

Comment:
I can't accept this, it looks like a session cache poisoning footgun. Please use a cache keyed by `$uid`.

Reply:
Örgs, you are right.

Reply:
This possibly explains some weird behaviour I observed.

Reply:
It is changed into an array.